### ISSUE TITLE
Sort idea spaces from lowest to highest number

### ISSUE THREAD

**Issue by rittermo** [liqd/aula#513]
Idea spaces in the Idea space overview should start with the school wide idea space and then be sorted by from the lowest number to the highest (meaning the school class numbers of 06c should come before school class 10c). Once in productive use, there will be idea spaces that have no numbers but just alphabetic names. No sure how they would be sorted?

**Comment by fisx** [comment]
we should have some heuristic: split up the string into a list of `Either String Int` and sort that:

``haskell
"10a" => [Right 10, Left a]
"ballroom dance 101" => [Left "ballroom dance ", Right 101]
```

(or perhaps split the strings further up into words?  not sure which one's better there.)

**Comment by fisx** [comment]
fixed in #523.
